di-demo/client/src/components/Article/ArticleStyle.ts

```diff
@@ -0,0 +1,37 @@
+import { themeColor, themeSpacing } from '@datapunt/asc-ui';
+import styled from 'styled-components';
+
+import Article from './Article';
+
+const ArticleStyle = styled(Article)`
+   border-right: 2px solid black;
+   padding-bottom: 16px;
+   margin-bottom: 28px;
+
+   a {
```

Comment:
Deze element references zijn wel wat rigid (h2, img in a) imho, waardoor een aanpassing in de structuur van het component sneller leidt tot styles die niet gezet worden

Reply:
ik begrijp je vraag niet : wat zou je anders willen?